src/modal/stories/index.stories.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component, Fragment } from 'react';
+import styled from 'styled-components';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import Button from '../../button';
+import Modal from '..';
+
+class ModalExample extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVisible: false
+    };
+  }
+
+  hide = () => {
+    this.setState({ isVisible: false });
+  }
+
+  show = () => {
+    this.setState({ isVisible: true });
+  }
+
+  render() {
+    return this.props.children({
+      hide: this.hide,
+      isVisible: this.state.isVisible,
+      show: this.show,
+    });
+  }
+}
+
+storiesOf('Modal', module)
+  .add('Basic Modal', () => (
+    <Fragment>
+      <ModalExample>
+        {
+          ({ hide, isVisible, show }) => (
+            <Fragment>
+              <Modal
+                  isVisible={isVisible}
+                  onClose={hide}
+                  textPrimary="Confirm"
+                  textSecondary="Cancel"
+                  textTitle="Metallica, Master of Puppets - March 3, 1986">
+                <p>
+                  March 3, 1986
+                </p>
+                <a href="https://openlattice.com">
+                  https://openlattice.com
+                </a>
+              </Modal>
+              <Button onClick={show}>
+                Show Modal
+              </Button>
+            </Fragment>
+          )
+        }
+      </ModalExample>
+      <br />
```

Comment:
Do you think it would be better to put each of these examples in their own "chapter" (dunno if that's what they're actually called)?

> .add('positioning', () => {...}
> .add('action appearance', () => {...}
> .add('closing', () => {...}
> .add('custom', () => {...}

There are lots of ways to use this component and as we add more examples down the line, it would be useful to have these examples be searchable in the sidebar and more easily accessible.

Reply:
Done.